floor_generator/test/test_utils.dart

```diff
@@ -62,7 +62,7 @@ Future<DartType> getDartTypeWithPerson(String value) async {
   return resolveSource(source, (item) async {
     final libraryReader =
         LibraryReader((await item.findLibraryByName('test'))!);
-    return (libraryReader.allElements.first as PropertyAccessorElement)
+    return (libraryReader.allElements.elementAt(1) as PropertyAccessorElement)
```

Comment:
interesting that the position changed 🤷 do you know what's at the first position now?

Reply:
Seems to always start with the element you use as the `LibraryReader` constructor's parameter 👌